tests/RegressionTests_hera.intel.log

```diff
@@ -4483,87 +1089,9 @@ Checking test 081 cpld_debug results ....
-Test 083 datm_restart_cfsr PASS
-
-
-baseline dir = /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20201223/INTEL/datm_control_gefs
-working dir  = /scratch1/NCEPDEV/stmp2/Denise.Worthen/FV3_RT/rt_257297/datm_control_gefs
-Checking test 084 datm_control_gefs results ....
- Comparing RESTART/MOM.res.nc .........OK
- Comparing RESTART/iced.2011-10-02-00000.nc .........OK
- Comparing RESTART/DATM_GEFS.cpl.r.2011-10-02-00000.nc .........OK
-Test 084 datm_control_gefs PASS
-
-
-baseline dir = /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20201223/INTEL/datm_bulk_cfsr
-working dir  = /scratch1/NCEPDEV/stmp2/Denise.Worthen/FV3_RT/rt_257297/datm_bulk_cfsr
-Checking test 085 datm_bulk_cfsr results ....
- Comparing RESTART/MOM.res.nc .........OK
- Comparing RESTART/iced.2011-10-02-00000.nc .........OK
- Comparing RESTART/DATM_CFSR.cpl.r.2011-10-02-00000.nc .........OK
-Test 085 datm_bulk_cfsr PASS
-
-
-baseline dir = /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20201223/INTEL/datm_bulk_gefs
-working dir  = /scratch1/NCEPDEV/stmp2/Denise.Worthen/FV3_RT/rt_257297/datm_bulk_gefs
-Checking test 086 datm_bulk_gefs results ....
- Comparing RESTART/MOM.res.nc .........OK
- Comparing RESTART/iced.2011-10-02-00000.nc .........OK
- Comparing RESTART/DATM_GEFS.cpl.r.2011-10-02-00000.nc .........OK
-Test 086 datm_bulk_gefs PASS
-
-
-baseline dir = /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20201223/INTEL/datm_mx025_cfsr
-working dir  = /scratch1/NCEPDEV/stmp2/Denise.Worthen/FV3_RT/rt_257297/datm_mx025_cfsr
-Checking test 087 datm_mx025_cfsr results ....
- Comparing RESTART/MOM.res.nc .........OK
- Comparing RESTART/MOM.res_1.nc .........OK
- Comparing RESTART/MOM.res_2.nc .........OK
- Comparing RESTART/MOM.res_3.nc .........OK
- Comparing RESTART/iced.2011-10-02-00000.nc .........OK
- Comparing RESTART/DATM_CFSR.cpl.r.2011-10-02-00000.nc .........OK
-Test 087 datm_mx025_cfsr PASS
-
-
-baseline dir = /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20201223/INTEL/datm_mx025_gefs
-working dir  = /scratch1/NCEPDEV/stmp2/Denise.Worthen/FV3_RT/rt_257297/datm_mx025_gefs
-Checking test 088 datm_mx025_gefs results ....
- Comparing RESTART/MOM.res.nc .........OK
- Comparing RESTART/MOM.res_1.nc .........OK
- Comparing RESTART/MOM.res_2.nc .........OK
- Comparing RESTART/MOM.res_3.nc .........OK
- Comparing RESTART/iced.2011-10-02-00000.nc .........OK
- Comparing RESTART/DATM_GEFS.cpl.r.2011-10-02-00000.nc .........OK
-Test 088 datm_mx025_gefs PASS
-
-
-baseline dir = /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20201223/INTEL/datm_debug_cfsr
-working dir  = /scratch1/NCEPDEV/stmp2/Denise.Worthen/FV3_RT/rt_257297/datm_debug_cfsr
-Checking test 089 datm_debug_cfsr results ....
- Comparing RESTART/MOM.res.nc .........OK
- Comparing RESTART/iced.2011-10-01-21600.nc .........OK
- Comparing RESTART/DATM_CFSR.cpl.r.2011-10-01-21600.nc .........OK
-Test 089 datm_debug_cfsr PASS
+Test 020 cpld_debugfrac PASS
```

Comment:
It looks to me only cpl tests have been run on with hera.intel?

Reply:
That was my test run; if I'm working on things that change baselines I will usually make/verify a baseline during testing and commit the log it so I have a point to go back to if something doesn't work later.